tests/Homely.AspNetCore.Mvc.Helpers.Tests/DateTimeConverterTests/ReadTests.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Text;
+using System.Text.Json;
+using Shouldly;
+using Xunit;
+
+namespace Homely.AspNetCore.Mvc.Helpers.Tests
+{
+    public class ReadTests
+    {
+        [Fact]
+        public void GivenAValidDateTimeString_Read_ReturnsAValidDateTime()
+        {
+            // Arrange.
+            var converter = new DateTimeConverter("yyyy"); // dateTimeFormat is ignored, in this Read(..) method.
+
+            var dateTime = new DateTime(2000, 1, 2, 3, 4, 5);
+            var json = JsonSerializer.Serialize(dateTime);
+            ReadOnlySpan<byte> utf8Bom = Encoding.UTF8.GetBytes(json);
+            var reader = new Utf8JsonReader(utf8Bom);
+
+            // First char (in the reader) is a token == None because we're at the start.
+            // So we need to move to the start of the actual json data.
+            // REF: https://stackoverflow.com/a/59039551/30674
+            reader.Read();

```

Comment:
such a complex arrange 😞 . assuming there's no easier way?

Reply:
well, i could make it one line, but it would be ... weird.

this is what i need to do, in reverse...
- new Utf8JR that accepts a `byte[]`
- create a byte[] from some json that represents some *ISO 8601* datetime
- serialize a datetime
- create a new datetime

- so if i knew the json string of some datetime, i can cut out the last 2 steps (above)
- if i knew the byte code of a json string, i can cut out the last 3 steps (above).

so .. nope :( It mainly helps with support/maintenance of this method and test, IMO.